src/idl_parser.cpp

```diff
@@ -1314,6 +1350,38 @@ CheckedError Parser::ParseSingleValue(Value &e) {
   return NoError();
 }
 
+void Parser::InitializeNestedFlatbufferParser(Parser& nestedParser, Value *const nested) {
+  assert(nested);
```

Comment:
I guess I'll go and write a unit test for multiple layers of nested flatbuffers, just to see how far this can be pushed.
In the end, I agree that it does not make much sense, unless someone plans to make an asset database out of nested flatbuffer assets.